src/main/views/features/common/name/capture-name.njk

```diff
@@ -5,7 +5,7 @@
 
 {% block main_content %}
 
-<main>
+
```

Comment:
Can you double check formatting/indentation after removing the main tags on all the pages please?